autocomplete/complete.go

```diff
@@ -94,27 +96,69 @@ func getPotentialTarget(prefix string) ([]string, error) {
 	return potentials, nil
 }
 
-func getPotentialPaths(prefix string) ([]string, error) {
-	if prefix == "." {
-		return []string{"./", "../"}, nil
+func lookupUsers(prefix string) ([]string, error) {
+	u, err := user.Current()
+	if err != nil {
+		return nil, err
 	}
-	currentUser, err := user.Current()
+
+	lPath := strings.Split(u.HomeDir, "/")
```

Comment:
Wonder if we should use `filepath.Separator` for all the `"/"`

Reply:
the filepath.Separator is an int32, do you prefer to have a var to replace the "/"? something like this 
```
var pathSeparator = fmt.Sprintf("%c", os.PathSeparator)

```

Reply:
Oh, interesting. Didn't realize it's int32.

I think this should be fine as it is. We anyway don't support auto-complete on non-unix systems.